zulip/integrations/bridge_with_matrix/matrix_bridge.py

```diff
@@ -192,11 +198,49 @@ def read_configuration(config_file: str) -> Dict[str, Dict[str, str]]:
 
     return {section: dict(config[section]) for section in config.sections()}
 
+def write_sample_config(target_path: str) -> None:
+    if os.path.exists(target_path):
+        raise Bridge_Exception("Path '{}' exists; not overwriting existing file.".format(target_path))
+
+    sample_dict = OrderedDict((
+        ('matrix', OrderedDict((
+            ('host', 'https://matrix.org'),
+            ('username', 'username'),
+            ('password', 'password'),
+            ('room_id', '#zulip:matrix.org'),
+        ))),
+        ('zulip', OrderedDict((
+            ('email', 'glitch-bot@chat.zulip.org'),
+            ('api_key', 'aPiKeY'),
+            ('site', 'https://chat.zulip.org'),
+            ('stream', 'test here'),
+            ('topic', 'matrix'),
+        ))),
+    ))
+
+    sample = configparser.ConfigParser()
+    sample.read_dict(sample_dict)
+    with open(target_path, 'w') as target:
+        sample.write(target)
+
 def main() -> None:
     signal.signal(signal.SIGINT, die)
     logging.basicConfig(level=logging.WARNING)
 
-    options = parse_args()
+    parser = generate_parser()
+    options = parser.parse_args()
+
+    if options.sample_config:
+        try:
+            write_sample_config(options.sample_config)
+        except Bridge_Exception as exception:
+            sys.exit(exception)
+        print("Wrote sample configuration to '{}'".format(options.sample_config))
+        sys.exit(0)
+    elif not options.config:
+        print("Options required: -c or --config to run, OR --write-sample-config.")
```

Comment:
I think we should update the docs with this method. The README.md, [Matrix integration doc](https://github.com/zulip/zulip/blob/master/templates/zerver/integrations/matrix.md) and [IRC integration doc](https://github.com/zulip/zulip/blob/master/templates/zerver/integrations/irc.md)